packages/platforms/delta/armel/arm-delta-ag6248c/README.md

```diff
@@ -0,0 +1,25 @@
+#How to run ONL in DELTA AG6248C board
+
+For the first step, it only support install the ONL to the USB and boot up.
```

Comment:
Is the information in this README still accurate?

Reply:
the comments is updated, please check it in the commit b1eeddcc3186b27c5066977cfd3687c061bca952